app/models/test_track/fake/split.rb

```diff
@@ -0,0 +1,14 @@
+class TestTrack::Fake::Split
+  include Singleton
+
+  def split_details
+    @split_details ||= _split_details
+  end
+
+  private
+
+  def _split_details
+    split_hash = TestTrack::Fake::Schema.instance.split_hash
+    { "name" => "banner_color" }.merge(split_hash["banner_color"].except("weights"))
```

Comment:
going along with the comment i left elsewhere about not putting the split details in the schema.yml file, i think that we should just have this return fake data just like we do in the `fake_instance_attributes` method.

i am also noticing that this fake doesn't include any of the details about the variants in this split. is that an intentional omission?